src/marks/axis.d.ts

```diff
@@ -34,7 +37,7 @@ export interface GridOptions extends GridScaleOptions {
    * Plot.gridY({x: (y) => aapl.find((d) => d.Close >= y)?.Date, anchor: "right"})
    * ```
    */
-  anchor?: "top" | "right" | "bottom" | "left";
+  anchor?: "top" | "right" | "bottom" | "left" | "inline";
```

Comment:
This is a bug (both in the docs and the code). The grid mark doesn’t support the *inline* orientation.

Reply:
I guess we can allow it here in the TypeScript declarations (just like we allow the *right* orientation for the axisX in the TypeScript declarations, even though it isn’t allowed at runtime). I’ll fix it at runtime though.